pkg/scheduler/framework/plugins/nodevolumelimits/utils.go

```diff
@@ -29,6 +29,21 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 )
 
+type wantStatusError struct {
```

Comment:
Let's remove to `pkg/scheduler/testing`,  a new utils.go also LGTM. For this is for testing only and we need it in other plugin packages.

Reply:
Also tests, it's simple, but we may still need one.

Reply:
OK, I will do it. Sorry, I'm too busy these days.